subprojects/GOPATH/src/github.com/microsoft/hivedscheduler/pkg/algorithm/types.go

```diff
@@ -34,12 +35,12 @@ type (
 	CellPriority int32
 )
 
-type CellRequest struct {
-	VC            api.VirtualClusterName
-	ReservationId api.ReservationId
-	Chain         CellChain
-	Level         CellLevel
-	Priority      CellPriority
+type schedulingRequest struct {
+	vc            api.VirtualClusterName
+	reservationId api.ReservationId
+	chain         CellChain
+	podGpuNumbers map[int32]int32
+	priority      CellPriority
```

Comment:
The podGpuNumbers must be the same as "pods in the same affinity group", if so why not directly use latter?

Reply:
You're right. But this schedulingRequest is an input to a scheduling algorithm. It seems that a scheduling algorithm does not have to be aware of the concept of affinity group. If you think it's better to rename, I'm ok :)

---
In reply to: [332507384](https://github.com/microsoft/pai/pull/3684#discussion_r332507384) [](ancestors = 332507384)

Reply:
Let's rename it. podGpuNumbers has less meaning than affinityGroup

---
In reply to: [334217302](https://github.com/microsoft/pai/pull/3684#discussion_r334217302) [](ancestors = 334217302,332507384)